news/URBBDC-3229.feature

```diff
@@ -0,0 +1,2 @@
+- Updated translations for workflow states in the housing procedure.
```

Comment:
Changelog entry should not contain `- ` otherwise, the release will be refused by PyPI